sem/runner.py

```diff
@@ -327,24 +329,31 @@ def run_simulations(self, parameter_list, data_folder, stop_on_errors=False):
             if return_code != 0:
                 with open(stdout_file_path, 'r') as stdout_file, open(
                         stderr_file_path, 'r') as stderr_file:
-                    complete_command = sem.utils.get_command_from_result(self.script, current_result)
-                    complete_command_debug = sem.utils.get_command_from_result(self.script, current_result, debug=True)
-                    error_message = ('\nSimulation exited with an error.\n'
-                                     'Params: %s\n'
-                                     'Stderr: %s\n'
-                                     'Stdout: %s\n'
-                                     'Use this command to reproduce:\n'
-                                     '%s\n'
-                                     'Debug with gdb:\n'
-                                     '%s'
-                                     % (parameter,
-                                        stderr_file.read(),
-                                        stdout_file.read(),
-                                        complete_command,
-                                        complete_command_debug))
+                    common_error_message = ('Params: %s\n'
+                                            'Stderr: %s\n'
+                                            'Stdout: %s\n'
+                                            % (parameter,
+                                               stderr_file.read(),
+                                               stdout_file.read()))
+                    if return_code == SIGKILL_CODE:
```

Comment:
May be worth it to always print `return_code` when it's not 0, even when it's not exactly -9. Can you add this information to the `common_error_message`?

Reply:
Added